packages/nuxt/test/fixtures/basic/app.vue

```diff
@@ -0,0 +1,3 @@
+<template>
+  <OnyxHeadline is="h1">Hello Onyx</OnyxHeadline>
```

Comment:
```suggestion
  <OnyxHeadline is="h1">Hello onyx</OnyxHeadline>
```

onyx is written in lowercase :D